docs/postman/Smart Village API.postman_collection.json

```diff
@@ -72,7 +72,7 @@
 				],
 				"body": {
 					"mode": "raw",
-					"raw": "{\n  \"account\": {\n    \"username\": \"kim.doe\",\n    \"email\": \"kim.doe@example.com\",\n    \"status\": 1,\n    \"tagsField\": [\n      \"Administration\",\n      \"Support\",\n      \"HumHub\"\n    ],\n    \"contentcontainer_id\": 5\n  },\n  \"profile\": {\n    \"firstname\": \"John\",\n    \"lastname\": \"Doe\",\n    \"title\": \"Test user\",\n    \"gender\": \"male\"\n\n  },\n  \"password\": {\n    \"newPassword\": \"SuperSecretPassword\", \n \"newPasswordConfirm\": \"SuperSecretPassword\" \n  }\n}",
+					"raw": "{\n  \"account\": {\n    \"username\": \"kim.doe\",\n    \"email\": \"kim.doe@example.com\",\n    \"status\": 1,\n    \"tagsField\": [\n      \"Administration\",\n      \"Support\",\n      \"HumHub\"\n    ],\n    \"contentcontainer_id\": 5\n  },\n  \"profile\": {\n    \"firstname\": \"John\",\n    \"lastname\": \"Doe\",\n    \"title\": \"Test user\",\n    \"gender\": \"male\"\n\n  },\n   \"legal\":{\n       \"dataPrivacyCheck\" : true\n  },\n  \"password\": {\n    \"newPassword\": \"SuperSecretPassword\", \n \"newPasswordConfirm\": \"SuperSecretPassword\" \n  }\n}",
```

Comment:
i updated our staging system and tried to do a request. i always get the HTML page returned when calling `/auth/register` now, no matter what data i send with the body params. is this related to the changes somehow. i could not figure out.

Reply:
maybe because we are calling an endpoint without bearer token, the privacy policy kicks in?

Reply:
i tried different things and even after registering worked without HTML page, then `GET /mail` returns the HTML page

Reply:
> i updated our staging system and tried to do a request. i always get the privacy policy HTML page returned when calling `POST /auth/register` now, no matter what data i send with the body params. is this related to the changes somehow. i could not figure out.

Hi @donni106 
This problem we are not facing in our side we  easily created user with privacy policy condition.
![image](https://user-images.githubusercontent.com/103625795/169637955-820afb79-0a8f-41e9-a1a5-e6c189506d14.png)


> maybe because we are calling an endpoint without bearer token, the privacy policy kicks in?

Yes, we are calling this endpoint without bearer token and also privacy policy code will executed depending on the status of dataPrivacyCheck key but it should not show a HTML page.


> i tried different things and even after registering worked without HTML page, then `GET /mail` returns the HTML page

And even /mail endpoint works fine.
![image](https://user-images.githubusercontent.com/103625795/169638441-0d0fccc3-bc75-425e-b17a-1a3af7bb9dac.png)

Reply:
and what if you pass no `legal` key or `dataPrivacyCheck : false`?